build.sh

```diff
@@ -205,7 +205,10 @@ if [[ ! -d "$Z3_DIR" ]]; then
     export Z3_DIR="$SVFHOME/$Z3Home"
 fi
 
-export PATH=$LLVM_DIR/bin:$PATH
+# Add LLVM & Z3 to $PATH and $LD_LIBRARY_PATH (prepend so that selected instances will be used first)
+export PATH=$LLVM_DIR/bin:$Z3_DIR/bin:$PATH
+export LD_LIBRARY_PATH=$LLVM_DIR/lib:$Z3_BIN/lib:$LD_LIBRARY_PATH
+
```

Comment:
There is no need to export since the end of `build.sh` will call `setup.sh` to establish the environmental variables.

Reply:
Alright, I'll remove those references.